### ISSUE TITLE
multi-image.py does this...?

### ISSUE THREAD

**Issue by myangelaku** [andrew12/osr-stuff#11]
![vlc_Z3WPUo1E04](https://user-images.githubusercontent.com/28674643/81816400-90946c80-952b-11ea-9391-128074175146.png)

**Comment by andrew12** [comment]
I need more context, how are you running it?

**Comment by andrew12** [comment]
Also I haven't worked on this project in a few years but I've been thinking about picking it back up lately

**Comment by myangelaku** [comment]
> Also I haven't worked on this project in a few years but I've been thinking about picking it back up lately

Firstly, I'd really appreciate it. It's a really good tool (I used multi_render.py before and just recorded it)


> I need more context, how are you running it?

aaa I'm sorry
I'm running it on my windows machine by typing "python multi_image foldername" (foldername includes some replays, the .osu file and the song) in a command shell in the folder, then using the ffmpeg script to render it to a video

The image is a screenshot of the video I get.

**Comment by myangelaku** [comment]
bump I guess :/

**Comment by andrew12** [comment]
Hi @myangelaku sorry for taking so long to reply. The folder needs to have a bunch of replays for a particular map, as well as an mp3 for the song (only if using multi_render though, but you need it to render it into a video with ffmpeg too).

**Comment by myangelaku** [comment]
Yes, I know, however the cursortrails don't disappear when using multi_image
